test/with_dummyserver/test_proxy_poolmanager.py

```diff
@@ -43,6 +43,16 @@
 from .. import TARPIT_HOST, requires_network
 
 
+def assert_is_verified(pm: ProxyManager, proxy: bool, host: bool) -> None:
```

Comment:
Thoughts on making the `proxy` and `host` values keyword-only so they self-document within the test case?

Reply:
fixed, looks much better now.

(additionally renamed `host` to `target`)